tests/framework/helpers/class-helper-donor.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * Class Give_Helper_Donor.
+ *
+ * Helper class to create a donor.
+ */
+class Give_Helper_Donor extends Give_Unit_Test_Case {
+
+	/**
+	 * Create a donor.
+	 *
+	 * @since 2.0
+	 *
+	 * @param array $donor_args
+	 *
+	 * @return int|Give_Donor
+	 */
+	public static function create_simple_payment( $donor_args = array() ) {
```

Comment:
@raftaar1191 This function should be `create_donor` and not `create_simple_payment`